site/webapp/src/main/webapp/static/js/jquery-hippo-validate.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2014 Hippo B.V. (http://www.onehippo.com)
+ */
+
+jQuery.validateExtend({
```

Comment:
@thojava does Bloomreach suggest using jQuery?

Reply:
@duncanheron I don't think Bloomreach have any preference on this. But the Enterprise Form plugin sample code using Jquery so I followed that.

Reply:
@duncanheron I don't think Bloomreach have any preference on this. But the Enterprise Form plugin sample code using Jquery so I followed that.

Reply:
Ok if the form plugin uses that convention thats fine - leave this as is. 👍